CONTRIBUTING.md

```diff
@@ -34,6 +34,58 @@ $ grunt docs
 $ grunt live
 ```
 
-When using the server provided by `grunt live`, the Strand documentation can be accessed at http://localhost:8000/docs/
+When using the server provided by `grunt live`, the Strand documentation can be accessed at `http://localhost:8000/docs/`
+
+## <a name="issue"></a> Found an Issue?
+If you find a bug in the source code or a mistake in the documentation, help us by
```

Comment:
Maybe not in this section but i think we need some clarity around 'create an issue regardless of pr' to avoid the submit a bad fix, pr closed no issue to track it scenario